src/Filesystem/File.php

```diff
@@ -455,7 +455,7 @@ public function exists(): bool
     public function perms()
     {
         if ($this->exists()) {
-            return substr(sprintf('%o', fileperms($this->path)), -4);
+            return substr(sprintf('%o', fileperms($this->path)), -3);
```

Comment:
Earlier the method return string of form `0xxx` but that was causing tests to fail as expected value generated used `deoct()` was of form `xxx` without th preceding `0`.

Reply:
What about `sprintf('%03o', fileperms($this->path) & 0777)`?